### DISCUSSION TITLE
Aqara WSDCGQ11LM funziona per 10 minuti

### DISCUSSION THREAD

**Discussion by Stomodine** [indomus/forum#1354] [GENERALE - Discussioni generiche]
Ciao

Nessun problema nel pairing del sensore a Zigbee2mqtt su Conbee II e per i primi 10 minuti tutto funziona correttamente. I problemi insorgono quando la lettura della temperatura raggiunge un valore "stabile" e da lì non invia più niente (vedi sotto,raggiunti i 6 gradi non aggiorna più).

Ho già provato a cambiare la batteria, ad associarlo direttamente al coordinatore ma niente da fare

![immagine](https://user-images.githubusercontent.com/96614159/150360307-3f5ea989-492e-4bbc-9388-abc5ab4ae8a1.png)

**Comment by indomus** [comment]
Il punto è che se la temperatura non cambia di almeno un decimo di grado, lui non invia alcun dato. A guardare il grafico pare proprio ci sia un livellamento, quindi il comportamento pare normale. Prova a farsi subire uno sbalzo di temperatura e verifica.

**Reply by Stomodine** [reply]
ho dimenticato di dire che anche provando a spostare il sensore fuori casa non cambia la lettura. sembra quasi che una volta livellata la temperatura entri in sleep mode e da lì non si schioda più ( a meno di rifare il pairing ovviamente)

**Reply by KC-inDomus** [reply]
molto molto strano. se lo hai già resettato una volta, lo hai riassociato e continua così, boh è strano forte, io ne ho 5 e nessuno si comporta così.

**Reply by Stomodine** [reply]
aggiungo: quando è associato e funzionante, premendo brevemente il tastino fa un paio di lampeggi rapidi che sta a significare la comunicazione con il coordinator. quando smette di funzionare invece , premendo il tasto fa un lampeggio singolo che è lo stesso comportamento di quando non è ancora associato

Riativando il "permit join" e tendendo premuto, si riassocia senza problemi e torna a funzionare per un tot di minuti. Tu cosa usi? io Z2M e conbee II su rasp 4

**Reply by snakuzzo** [reply]
Non devi fare ogni volta il permit join. Il device non è che si "disassocia" in automatico.
Questi sono device che vanno in deep sleep e, come diceva @KC-inDomus, si svegliano quando c'è una variazione di temperatura significante. Ovviamente quando premi brevemente il tastino li svegli e lì parte la comunicazione con z2m che poi pubblica sul broker MQTT i vari stati del device.
Prova a verificare le impostazioni lato zigbee2mqtt per capire se c'è qualcosa di specifico da impostare per quel device. [QUI](https://www.zigbee2mqtt.io/devices/WSDCGQ11LM.html) la pagina dedicata.
Inoltre prova ad impostare un livello di log più alto (leggi [QUI](https://www.zigbee2mqtt.io/guide/usage/debug.html#enabling-logging)) per capire se è zigbee2mqtt che non pubblica i valori inviati dal device o se proprio non gli arrivano i messaggi dal device.

**Reply by Stomodine** [reply]
ciao

non so se il dispositivo sta andando in deep sleep ma il fatto è che non invia più i dati dopo un tot. l'aggiornamento viene fatto secondo quanto dice il produttore
![immagine](https://user-images.githubusercontent.com/96614159/150645017-b07e107a-8230-4506-aab7-e6d1157c4bb1.png)

il problema è che se non attivo il "permit join" il sensore non mi invia più le letture (come se non si svegliasse più). Ho una mezza idea che sia un problema combinato di batteria ( ho provato a rimettere la sua originale e ha funzionato per 1gg intero salvo poi morire di nuovo) e di portata segnale.

mi sapresti sugerire un'automazione che mi notifica se il dispositivo non comunica nessuno stato per 2 ore di fila? cioè,so fare quasi tutto ma mi manca la parte del conteggio ore e del messaggio mqtt da monitorare

**Comment by Stomodine** [comment]
Lacscio le mie considerazioni finali : il problema è un mix di batteria inefficiente e segnale Zigbee non eccellente.

Quindi se dovesse capitarvi ricordate di mettere batterie MOLTO affidabili e avere un segnale ECCELLENTE (io ho risolto mettendo il ripetitore dell'Ikea)

**Reply by KC-inDomus** [reply]
ti dirò, per mia esperienza personale su quel sensore il segnale è un problema quando è veramente ridotto quasi a 0, per il resto comunicare comunica. sulla batteria non so.

**Reply by Stomodine** [reply]
grazie per la info...nel mio caso ero intorno ai 100 quindi sulle prime non me ne ero troppo preoccupato.

Magari con le nuove versioni è cambiato qualcosa? boh,tiro ad indovinare
